src/dotnet-gcmon/readme.txt

```diff
@@ -0,0 +1,23 @@
+How to make a global .NET CLI tool to monitor garbage collection in real time?
```

Comment:
Is this file redundant now that the [README.md](https://github.com/Maoni0/realmon#how-to-generate-the-global-net-cli-tool-dotnet-gcmon) contains the same information?

Reply:
Good point: I thought I deleted it in a commit of the original PR.

Reply:
✅